adcm_aio_client/core/objects/cm.py

```diff
@@ -36,7 +38,50 @@ def get_own_path(self: Self) -> Endpoint:
         return ("adcm",)
 
 
-class Bundle(Deletable, InteractiveObject): ...
+class License(InteractiveObject): ...
+
+
+class Bundle(Deletable, RootInteractiveObject):
+    PATH_PREFIX = "bundles"
+
+    @property
+    def name(self: Self) -> str:
+        return str(self._data["name"])
+
+    @property
+    def display_name(self: Self) -> str:
+        return str(self._data["display_name"])
+
+    @property
+    def version(self: Self) -> str:
+        return str(self._data["version"])
+
+    @property
+    def edition(self: Self) -> Literal["community", "enterprise"]:
+        return self._data["edition"]
+
+    @property
+    def signature_status(self: Self) -> Literal["invalid", "valid", "absent"]:
+        return self._data["signatureStatus"]
+
+    @property
+    def _type(self: Self) -> Literal["cluster", "provider"]:
+        return self._data["mainPrototype"]["type"]
+
+    @CachedProperty
```

Comment:
please, use as in here https://github.com/arenadata/adcm-aio-client/pull/18/files#diff-9af600651db7e4b5e12436271f04d1c653622b252006ff3607e0ba34e19824a6R5